src/openssl_lib.cpp

```diff
@@ -242,6 +242,15 @@ int OpenSSLLib::SSL_EVP_DigestSignInit(EVP_MD_CTX* ctx,
     return EVP_DigestSignInit(ctx, pctx, type, e, pkey);
 }
 
+int OpenSSLLib::SSL_EVP_DigestSign(EVP_MD_CTX* ctx,
```

Comment:
hmm functions in this file are auto-generated using the `add_function.py` script. Because the functions in here are simply calling the appropriate openssl functions, it might be a good idea to keep the openssl parameter-names here (so they match the openssl documentation).